lib/fleiss/worker.rb

```diff
@@ -62,34 +60,37 @@ def run_cycle
 
     batch.each do |job|
       @pool.post do
-        Fleiss.backend.wrap_perform { perform(job) }
+        perform(job)
```

Comment:
Can you please revert this, it was quite nice to have it here. If you want thread id and owner, just assign them here and pass them as vars to perform.

Reply:
Ok, let me have another go on cosmetics...